src/services/data/handlers/sink.ts

```diff
@@ -108,175 +109,120 @@ export const seatool: Handler = async (event) => {
   }
 };
 
-export const onemac: Handler = async (event) => {
-  const oneMacRecords: (
-    | OnemacTransform
-    | OnemacRecordsToDelete
-    | OnemacLegacyTransform
-    | OnemacLegacyRecordsToDelete
-    | (ToggleWithdrawRaiEnabled & { id: string })
-    | RaiIssueTransform
-    | RaiResponseTransform
-    | RaiWithdrawTransform
-    | WithdrawPackageTransform
-  )[] = [];
+export const onemacDataTransform = (props: { key: string; value?: string }) => {
```

Comment:
@mdial89f  

As we discussed, this is an event transform mapping abstracted from original onemac sink.